static.config.js

```diff
@@ -327,6 +329,30 @@ const getOfficialDocumentCollectionData = async (page, instance, client) => {
   return { officialDocumentCollection: officialDocumentCollection };
 };
 
+const getOfficialDocumentPageData = (page, instance) => {
+  let officialDocumentPage = { ...page };
+
+  if (officialDocumentPage.document.filename.slice(-3) === 'pdf') {
+    officialDocumentPage.pdfSize = filesize(
+      officialDocumentPage.document.fileSize,
+    ).replace(' ', '');
+  }
+
+  officialDocumentPage.officialDocumentCollection = cleanOfficialDocumentPageCollections(
+    officialDocumentPage.officialDocumentCollection,
+  );
+
+  officialDocumentPage.contextualNavData = {
+    // parent: officialDocumentPage.officialDocumentCollection[0],
+    parent: instance.parent,
```

Comment:
Tori said: "Could you use the parent topic of the parent collection? (or first parent collection if it belongs to more than 1? Just thinking about options. It’s either that or the parent collection."

However, I don't think I can easily get the parent topic of the parent collection, I am going to see at dev design sync if she wants to go with the parent department, or with the official document collection.